tasks/gettext-checker.js

```diff
@@ -52,10 +52,27 @@ module.exports = function (grunt) {
         validatePoFile(templateFile, options.templateFile);
         validatePoFile(poFile, options.poFile);
 
+        var getMessageId = function (item) {
+            return item.msgid;
+        };
+
         var getPoKeys = function (items) {
-            return _.map(items, function (item) {
-                return item.msgid;
+            var keys = {
+                used: [],
+                obsolete: []
+            };
+            // Separate used keys from commented keys. Use two filter loops
+            // instead of one loop and pushing, because pushing isn't quick.
+            var usedItems = items.filter(function (item) {
+                return item.obsolete === false;
+            });
+            var obsoleteItems = items.filter(function (item) {
```

Comment:
I wrote both filters so I could manually compare to test them, so I thought "work is done, may as well keep it"